_docs/applications/postgres.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: guide
-title: StorageOS Docs - PostgreSQL
+title: PostgreSQL with StorageOS
```

Comment:
Don't change this, this is displayed in the tab title bar, so all guides should follow the same format of "StorageOS Docs - ".